### PR TITLE
Tennis Balls! (and donator stuff)

### PR DESCRIPTION
**Pull request by Michiyamenotehifunana**
:cl: Toriate
add: Added tennis balls! Now available in 7 different flavors! You can waggle them at people.
add: added right-click interactions for NEBs
imageadd: added tennis ball sprites
/:cl:

This PR will add tennis balls, available in 7 different variants and waggle-able at things! Each one (aside from the classic one) has minor quirks that stand out from the rest. For example, the red tennis ball goes 3x as fast as the others.

The tennis balls can be equipped in the head slot just like bread slices, as well as the ears and neck slot. They cannot go in the mask slot due to code limitations. They have complete sets of mob sprites.

dreamseeker_2018-03-16_18-01-16

There are two tennis ball variants reserved for donors, namely the rainbow one and Izzy's. The 7 variants are available in the loadout system.

In addition, this PR includes donator items (mine and Izzy's), and adds right-click interactions to NEBs.

### CODE REVIEW

**Comment by GuppyLaxx** [comment]
Hey, hey guys... balls, hueheuehueueheh

**Comment by NopeMcHaltman** [comment]
But what about rackets? :thinking:

File: modular_citadel/code/game/objects/items/balls.dm [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/obj/item/toy/tennis
+	name = "tennis ball"
+	desc = "A classical tennis ball. It appears to have faint bite marks scattered all over its surface."
+	icon = 'modular_citadel/icons/obj/balls.dmi'
+	icon_state = "tennis_classic"
+	lefthand_file = 'modular_citadel/icons/mob/inhands/balls_left.dmi'
+	righthand_file = 'modular_citadel/icons/mob/inhands/balls_right.dmi'
+	item_state = "tennis_classic"
+	alternate_worn_icon = 'modular_citadel/icons/mob/mouthball.dmi'
+	slot_flags = SLOT_HEAD
```
Comment (deathride58) [inline modular_citadel/code/game/objects/items/balls.dm]: If you want to make this equippable in more than just the head slot, check out how headphones handle it here https://github.com/Citadel-Station-13/Citadel-Station-13/blob/master/code/modules/clothing/ears/_ears.dm#L20

Comment (Michiyamenotehifunana) [reply modular_citadel/code/game/objects/items/balls.dm]: The thing is, it simply refuses to work properly in the mask slot. Even the icon_state breaks when it is placed in the mask slot for whatever reason.

**Review by Poojawa — APPROVED** [review]

**Review by deathride58 — APPROVED** [review]
@LetterJay Ready for merge whenever, if Toriate and Izzy both donated.

**Pull request merged**